cpp/arcticdb/version/test/test_symbol_list.cpp

```diff
@@ -153,14 +153,18 @@ TEST_P(SymbolListWithReadFailures, VersionMapSource) {
     ASSERT_THAT(symbols, UnorderedElementsAre(symbol_1, symbol_2, symbol_3));
 }
 
-INSTANTIATE_TEST_SUITE_P(, SymbolListWithReadFailures, Values(
+INSTANTIATE_TEST_SUITE_P(SymbolListWithReadFailuresSources, SymbolListWithReadFailures, Values(
```

Comment:
Why add this?

Reply:
leftover from debugging, this can be removed